libs/core/src/lib/panel/panel.service.ts

```diff
@@ -3,10 +3,13 @@ import { BehaviorSubject } from 'rxjs';
 
 @Injectable()
 export class PanelService {
-    expanded$: BehaviorSubject<boolean> = new BehaviorSubject(false);
+    isExpanded: boolean = false;
+    expanded$: BehaviorSubject<boolean> = new BehaviorSubject(this.isExpanded);
 
     /** Whether the Panel is expanded */
     updateExpanded(value: boolean) {
-        this.expanded$.next(value);
+        value === this.isExpanded ? this.isExpanded = !value : this.isExpanded = value
```

Comment:
I believe this can be simplified to:
`this.isExpanded = value === this.isExpanded ? !value : value;`

Reply:
@salarenko opened a PR that was merged into this branch